wfe/wfe_test.go

```diff
@@ -190,7 +190,7 @@ func (ra *MockRegistrationAuthority) NewCertificate(ctx context.Context, req cor
 }
 
 func (ra *MockRegistrationAuthority) UpdateRegistration(ctx context.Context, reg core.Registration, updated core.Registration) (core.Registration, error) {
-	if reg.Key != updated.Key {
+	if reg.Key.Key != updated.Key.Key {
```

Comment:
Ahh! That makes a lot of sense. Learned something new :-) Thanks!

I converted over to using `PublicKeysEqual` in https://github.com/letsencrypt/boulder/pull/2314/commits/f69f01dd84925aa6307eba9333ef47792546f81c